layers/+lang/latex/funcs.el

```diff
@@ -25,19 +25,33 @@
     (`lsp (spacemacs|add-company-backends
             :backends company-capf
             :modes LaTeX-mode))
-    (_    (when (configuration-layer/package-used-p 'company-auctex)
-            (spacemacs|add-company-backends
-              :backends
-              (company-auctex-macros)
-              company-auctex-symbols
-              company-auctex-environments
-              :modes LaTeX-mode))
-          (when (configuration-layer/package-used-p 'company-reftex)
-            (spacemacs|add-company-backends
-              :backends
-              company-reftex-labels
-              company-reftex-citations
-              :modes LaTeX-mode)))))
+    ('company-math (when (and (configuration-layer/package-used-p 'company-auctex) (configuration-layer/package-used-p 'company-math))
+                       (spacemacs|add-company-backends
+                         :backends
+                         company-auctex-environments
+                         company-auctex-macros
```

Comment:
This must be evaluated, to return the proper content.

Reply:
I do not understand this comment. Could you elaborate?

Reply:
Sure, I have just checked the other backends and it seems that not the value of `company-auctex-macros` is used but its return value i.e. in lisp terms `(company-auctex-macros)`.

Reply:
My mistake. I have changed the code. I checked and indeed company-auctex-macros produces a list of company backends. I got it wrong because I looked at [emacs wiki] (https://www.emacswiki.org/emacs/company-math) where the parentheses miss.